input/intro-notes/StructureDefinition-au-coverage-intro.md

```diff
@@ -2,3 +2,11 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- When populating the type using the preferred ValueSet :
+    - *Medicare* coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "PUBLICPOL"
+    - *DVA* coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "VET"
+    - *Private health insurance* coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "HIP"
+    - *Workcover* coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "WCBPOL"
+    - *Motor Accident* coverage  **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "AUTOPOL"
```

Comment:
The previous three lines all need the v3 Code System ActCode update like the first lines.

Reply:
I am unable to make changes to the resolution.